lib/ecto_enum_migration.ex

```diff
@@ -219,19 +219,11 @@ defmodule EctoEnumMigration do
     after_value = to_value(after_value)
 
     up_sql = "
-      UPDATE pg_catalog.pg_enum
-      SET enumlabel = #{after_value}
-      WHERE enumtypid = '#{type_name}'::regtype::oid
-        AND enumlabel = #{before_value}
-      RETURNING enumlabel;
+      ALTER TYPE #{type_name} RENAME VALUE #{before_value} TO #{after_value};
```

Comment:
@sam-levy Could you please add a warning to the doc that this function only works with postgres 10+?

Reply:
Sure. Warning added.